### ISSUE TITLE
Add experiment for additive coupling vs affine coupling as presented in the paper

### ISSUE THREAD

**Issue by Dayan-Zhanchi** [Lunkers/DD2412-Project#12]
Conduct experiments comparing the use of additive coupling and affine coupling.

All training conditions stay constant according to them.

- Need 3 random seeds for the 3 different runs to compute the average and std for NLL
- Same hyperparameters used as for comparison with RealNVP. Referring to appendix optimization details
- Measure time to see if affine coupling is faster than additive coupling
- Implement the 2 other permutation methods for the channels. Reverse and shuffle. 

Related to #11.

**Comment by Dayan-Zhanchi** [comment]
Dont need
